index.html

```diff
@@ -113,6 +113,24 @@
         li.appendChild( link );
         sidebar.appendChild( li );
 
+        // And benchmarks
+        var li = document.createElement( 'li' );
```

Comment:
I really don't want this to interfere in the PR I just added it for having in the main index, but if it is going to be a problem I rather remove it.